### ISSUE TITLE
Add support for UWP Compilation target

### ISSUE THREAD

**Issue by MrcSnm** [icculus/SDL_sound#29]
If you understand how to do that, I would appreciate if you could add that support, I'm developing a Xbox game using SDL_Sound.

I'll try my best at compiling it for UWP too, if I succeed before you,  I may PR, but I know that should be a kid's play for you

**Comment by MrcSnm** [comment]
How can I solve FATAL_ERROR,SDL2::SDL2 and/or SDL2::SDL2main don't seem to contain any kind of IMPORTED_IMPLIB* or IMPORTED_LOCATION*
CMake Error at C:/Users/Hipreme/Documents/Redist/SDL2-2.0.16/SDL2Config.cmake:94 (get_filename_component) ?

**Comment by MrcSnm** [comment]
I was able to generate SDL2Targets.cmake by:

cmake --configure sdlDir
cmake --build . --target install

It would be really enlightening having those build instructions somewhere. But I still need to find how to make SDL_Sound build now

**Comment by MrcSnm** [comment]
So I discovered that the problem seems to be that it dependends on kernel32.dll, if you guys have any idea on how to solve that, I would be happy

**Comment by sezero** [comment]
(sorry for keeping this in limbo for too long.)

Discovery of SDL for MSVC can be a pain, I guess.

As for kernel32.dll dependency: SDL_sound doesn't advertise such dependency,
so I'm guessing that it comes from the existing SDL2 devel package. Is linking to
kernel32.lib not supported in UWP??

I am clueless with UWP though -- this is for @icculus, I guess.

**Comment by icculus** [comment]
I'd have to check this on an actual Windows install, but as far as I know, SDL_sound itself has no dependency on kernel32 (SDL2 itself absolutely does, though).

Although maybe the playsound test program does, since it uses stdio and such?

**Comment by MrcSnm** [comment]
@sezero Nope, kernel32 is not available in the platforms UWP is meant to support.
AFAICT SDL_Sound wasn't able to work on the platform I tested. Idk maybe I wasn't able to fully integrate the SDL_Sound project on VS uwp project?

**Comment by icculus** [comment]
Closing this because it's very old and SDL3 dropped support for UWP outright, so we can't really fix this going forward anyhow.  :/
